docs/user/customization.md

```diff
@@ -27,6 +27,176 @@ The `manifests` target will render the manifest templates and output the result
+
+3. Run `cluster` target to create the cluster using the custom manifests.
+
+    ```console
+    $ openshift-install --dir no-taint-cluster create cluster
+    INFO Consuming "Openshift Manifests" from target directory
+    INFO Consuming "Master Machines" from target directory
+    INFO Consuming "Common Manifests" from target directory
+    INFO Creating cluster...
+    INFO Waiting up to 30m0s for the Kubernetes API...
+    ...
+    ```
+
+    Check that no control plane nodes registered with taints:
+
+    ```console
+    $ oc --config no-taint-cluster/auth/kubeconfig get nodes -ojson | jq '.items[] | select(.metadata.labels."node-role.kubernetes.io/master" == "") | .spec.taints'
+    null
+    ```
+
+    Check that the `02-master-kubelet` `MachineConfig` exists in the cluster:
+
+    ```console
+    oc --config no-taint-cluster/auth/kubeconfig get machineconfigs
+    NAME                                                        GENERATEDBYCONTROLLER        IGNITIONVERSION   CREATED
+    00-master                                                   3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    00-master-ssh                                               3.11.0-744-g5b05d9d3-dirty                     137m
+    00-worker                                                   3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    00-worker-ssh                                               3.11.0-744-g5b05d9d3-dirty                     137m
+    01-master-container-runtime                                 3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    01-master-kubelet                                           3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    02-master-kubelet                                                                        2.2.0             137m
+    01-worker-container-runtime                                 3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    01-worker-kubelet                                           3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    99-master-3c81ffa3-3b8d-11e9-ac1e-52fdfc072182-registries   3.11.0-744-g5b05d9d3-dirty                     133m
+    99-worker-3c83a226-3b8d-11e9-ac1e-52fdfc072182-registries   3.11.0-744-g5b05d9d3-dirty                     133m
+    master-55491738d7cd1ad6c72891e77c35e024                     3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    worker-edab0895c59dba7a566f4b955d87d964                     3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
+    ```
+
+[default-kubelet-service]: https://github.com/openshift/machine-config-operator/blob/master/templates/master/01-master-kubelet/_base/units/kubelet.yaml
+[ignition]: https://coreos.com/ignition/docs/latest/
```

Comment:
I think these references should stay at the bottom of the file (e.g. the `[ignition]` reference is (also?) consumed by the next `OS Customization (unvalidated)` section.

Reply:
moved ignition by mistake. fixed

but looks like other sections have their own anchors https://github.com/openshift/installer/pull/1331/files#diff-ae4bcbdd69693a4977c2088180a61927R19 so keeping the rest as-is.